agent/util-scripts/pbench-tool-meister-start

```diff
@@ -749,19 +1117,17 @@ def main(_prog, cli_params):
         "6. waiting for all successfully created Tool Meister processes"
         " to show up as subscribers"
     )
-    ret_val = wait_for_tds(to_client_chan, logger)
+    # ret_val = wait_for_tds(to_client_chan, logger)
+    ret_val = tool_data_sink.wait(to_client_chan, logger)
     if ret_val != 0:
-        if cli_params.redis_server is None:
-            # We created the Redis server, so we should clean it up.
-            return kill_redis_server(redis_pid, ReturnCode.TDSWAITFAILURE)
-        else:
-            return ReturnCode.TDSWAITFAILURE
+        # Clean up the Redis server if we created it.
+        return redis_server.kill(ReturnCode.TDSWAITFAILURE)
```

Comment:
Shouldn't this be under an `is_managed()` check?

Reply:
Fixed.